.azure-pipelines/templates/azdev_setup.yml

```diff
@@ -14,7 +14,7 @@ steps:
       chmod +x env/bin/activate
       . env/bin/activate
 
-      pip install -q azdev
+      pip install -q azdev==0.1.21
```

Comment:
no need to pin verison

Reply:
Unpin it. However, I think we should pin this version in case of potential bug in azdev causing the broken of our CI.

Reply:
We can unpin the azdev version, but run Azure CLI CI as a gate test when releasing new azdev to guarantee the azdev release package quality.